packages/api/db/migrations/30000022-update-cities-and-epci.js

```diff
@@ -0,0 +1,313 @@
+                        type: Sequelize.STRING,
+                        allowNull: false,
+                    },
+                },
+                { transaction },
+            ),
+        ])
+        // fill cities_2022
+            .then(() => parser(
+                fs.readFileSync(path.join(__dirname, '..', 'data', 'cities_2022.csv'), { encoding: 'utf8' }),
+                {
+                    headers: ['code', 'name', 'departementCode', 'regionCode', 'epciCode', 'arr', 'canton',
+                    ],
+                    separator: ';',
+                },
+            ))
+            .then(cities => cities.filter(city => city.COMPARENT === '' || city.COMPARENT !== city.code))
+            .then(cities => queryInterface.bulkInsert(
+                'cities_2022',
+                cities.map(city => ({
+                    code: city.code.length === 5 ? city.code : `0${city.code}`,
+                    name: city.name,
+                    fk_epci: city.epciCode,
+                    fk_departement: city.departementCode,
+                })),
+                { transaction },
+            ))
+        // fill epci_2022
+            .then(() => parser(
+                fs.readFileSync(path.join(__dirname, '..', 'data', 'epci_2022.csv'), { encoding: 'utf8' }),
+                {
+                    headers: ['code', 'label',
+                    ],
+                    separator: ';',
+                },
+            ))
+            .then(epcis => queryInterface.bulkInsert(
+                'epci_2022',
+                epcis.map(epci => ({
+                    code: epci.code,
+                    name: epci.label,
+                })),
+                { transaction },
+            ))
+        // fill mov_epci_2022
+            .then(() => parser(
+                fs.readFileSync(path.join(__dirname, '..', 'data', 'mov_epci_2022.csv'), { encoding: 'utf8' }),
+                {
+                    headers: ['oldCode', 'newCode',
+                    ],
+                    separator: ';',
+                },
+            ))
+            .then(epcis => queryInterface.bulkInsert(
+                'mov_epci_2022',
+                epcis.map(mov_epci => ({
+                    old_code: mov_epci.oldCode,
+                    new_code: mov_epci.newCode,
+                })),
+                { transaction },
+            ));
+
+        await Promise.all([
+            // updates on cities
+            queryInterface.sequelize.query(
+                'UPDATE cities SET fk_main = null WHERE fk_main IN (SELECT code FROM cities WHERE code NOT IN (SELECT code FROM cities_2022))',
+                {
+                    type: queryInterface.sequelize.QueryTypes.UPDATE,
+                    transaction,
+                },
+            ),
+            queryInterface.sequelize.query(
+                'DELETE FROM organizations WHERE fk_city IN (SELECT code FROM cities WHERE code NOT IN (SELECT code FROM cities_2022))',
```

Comment:
`DELETE FROM organizations WHERE fk_city NOT IN (SELECT code FROM cities_2022)` ?